src/Mjolnir.ts

```diff
@@ -506,13 +485,7 @@ export class Mjolnir {
 
     public async warnAboutUnprotectedPolicyListRoom(roomId: string) {
         if (!this.config.protectAllJoinedRooms) return; // doesn't matter
-        if (this.explicitlyProtectedRoomIds.includes(roomId)) return; // explicitly protected
-
-        const createEvent = new CreateEvent(await this.client.getRoomStateEvent(roomId, "m.room.create", ""));
```

Comment:
Yes, we remove this check because it's not an accurate way to determine whether Mjolnir should protect the policy room. Especially if another Mjolnir/user created it, or if some other Mjolnir is now responsible for that list.

I mentioned it in the description of the PR too

Reply:
We replace it with needing policy rooms to be explicitly protected in all circumstances and by showing you which policy rooms are protected distinctly when you use the status command (screenshot attached in PR too)

Reply:
We also still warn when you first watch a room and it is unprotected.